arangod/IResearch/IResearchDataStore.h

```diff
@@ -264,16 +278,18 @@ class IResearchDataStore {
   /// @brief the underlying iresearch data store
   //////////////////////////////////////////////////////////////////////////////
   struct DataStore {
-    IResearchDataStoreMeta
-        _meta;  // runtime meta for a data store (not persisted)
+    // runtime meta for a data store (not persisted)
+    IResearchDataStoreMeta _meta;
     irs::directory::ptr _directory;
-    basics::ReadWriteLock _mutex;  // for use with member '_meta'
+    // for use with member '_meta'
+    basics::ReadWriteLock _mutex;
     irs::utf8_path _path;
     irs::directory_reader _reader;
     irs::index_writer::ptr _writer;
     // the tick at which data store was recovered
     TRI_voc_tick_t _recoveryTick{0};
-    std::atomic_bool _inRecovery{false};  // data store is in recovery
+    // data store is in recovery
+    std::atomic_bool _inRecovery{false};
```

Comment:
can we move `_inRecovery` after `_failed` to avoid unnecessary padding

Reply:
Now arranging it like this:
```
struct DataStore {
  ...
  std::atomic_bool _inRecovery;
};

DataStore _dataStore;
std::atomic_bool _failed;
```
Ok?

Reply:
:+1: